week1/index.js

```diff
@@ -1,38 +1,30 @@
-import HTTP from 'http'
-import Path from 'path'
-
-import sendIndexHTML from './responses/sendIndexHTML'
-import sendPage2HTML from './responses/sendPage2HTML'
-import sendStylesCSS from './responses/sendStylesCSS'
-import sendText from './responses/sendText'
-
-const server = HTTP.createServer((request, response) => {
-	console.log(request.method, request.url)
-
-	switch (request.url) {
-	case '/':
-		sendIndexHTML(response)
-		break
-	case '/page2':
-		sendPage2HTML(response)
-		break
-	case '/styles.css':
-		sendStylesCSS(response)
-		break
-	default:
-		const extension = Path.extname(request.url)
-		if (extension === '') {
-			response.statusCode = 302
-			response.setHeader('Location', '/')
-		} else {
-			response.statusCode = 404
-			sendText(response, "File not found")
-		}
-	}
+const http = require('http');
+const port = 3000;
+let state = 10;
+let server = http.createServer((request, response)=>{
 	
-	response.end()
-})
-
-server.listen(3001)
+	
+	if (request.url === "/add"){
+		state +=1
+		response.write(`<h1>The state is : </h1><h2>${state}</h2>`)
 
-console.log('Server started')
\ No newline at end of file
+	}else if (request.url === "/remove"){
+		state -=1
+		response.write(`<h1>The state is : </h1><h2>${state}</h2>`)
```

Comment:
Without setting a Content-Type header, we're leaving it up to the browser to try and guess what type this is supposed to be. Can you add a Content-Type header so that it's explicitly declared that this will be html?

(Also note that all of your `response.write()` strings are only partial html, not full html. you should probably return a full html page (with `<html>`, `<head>`, and `<body>`)